docs/content/Theming/06_browser_color.md

```diff
@@ -0,0 +1,45 @@
+@ngdoc content
+@name Browser Colors
+@description
+
+<div class="layout_note">
+  <span>This feature is **mobile** only</span>
+</div>
+
+![browser color](https://cloud.githubusercontent.com/assets/6004537/18006666/50519c7e-6ba9-11e6-905b-c3751c20549c.png)
+
+Enables browser header coloring with [Material Design Colors](https://material.google.com/style/color.html) and the theming system, <br/> 
+For more information, please visit:<br/> https://developers.google.com/web/fundamentals/design-and-ui/browser-customization/theme-color.
```

Comment:
A bit nit-picky, but can you remove the `<br/>` tag and end the sentence with a `.` instead of a `,`?

Also, with the link being on it's own line, I feel like there should be one additional line between 11 and 12 to separate the sentences a bit.